modules/CodeTablesAdmin/src/client/AbstractCodeTablesAdmin.coffee

```diff
@@ -25,7 +25,8 @@ class window.AbstractCodeTablesAdminController extends AbstractFormController
 		"click .bv_cancelClear": "handleCancelClearClicked"
 		"click .bv_confirmClear": "handleConfirmClearClicked"
 
-	initialize: ->
+	initialize: (options) ->
+		@options = options
```

Comment:
Backbone upgrade no longer automatically attaches options so you need to do it yourself.

Version 1.1.0 https://backbonejs.org/#changelog

Reply:
I take back the version change argument for this.  We were on backbone 1.2.3 which already had this change.  Its unclear to me exactly why I needed to make this change to get this work.